sqldelight-gradle-plugin/src/test/fixtures/dots-in-folder/src/main/sqldelight/com.sample.test/Test.sq

```diff
@@ -0,0 +1,3 @@
+CREATE TABLE test (
```

Comment:
I think #40 is pretty urgent given that this is showing up. The solution I posted there (adding /sqldelight as a java source folder) is still the only thing that works. Thoughts on just doing that?